### DISCUSSION TITLE
Error on installing emoji-mart-react

### DISCUSSION THREAD

**Discussion by zeeshan-attentive** [missive/emoji-mart#729] [Q&A] [answered]
I installed emoji-mart-react using this command npm install --save emoji-mart @emoji-mart/data @emoji-mart/react. My application crashes and i get this error.

`Failed to compile.
./node_modules/emoji-mart/dist/module.js 1261:55
Module parse failed: Unexpected token (1261:55)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| // @ts-nocheck
| 
> const $6f57cc9cd54c5aaa$var$WindowHTMLElement = window?.HTMLElement ?? Object;
| class $6f57cc9cd54c5aaa$export$2e2bcd8739ae039 extends $6f57cc9cd54c5aaa$var$WindowHTMLElement {
|     static get observedAttributes() {`

I have searched a lot and i still can't seem to find a solution for this error.

**Comment by EtienneLem** [comment, marked as answer]
See #727

**Reply by zeeshan-attentive** [reply]
I reinstalled the emoji-mart-react using this command npm install --save emoji-mart @emoji-mart/data @emoji-mart/react and the problem still persists.

**Reply by EtienneLem** [reply]
Should be fixed in **v5.3.2**; it was just released. Make sure `emoji-mart` package is updated, not `@emoji-mart/react`

**Reply by zeeshan-attentive** [reply]
Yes. It is fixed now. Thank you so much.
